CHANGELOG.md

```diff
@@ -4,13 +4,12 @@ All notable changes to this project will be documented in this file. See [commit
 
 ## [1.1.0](https://github.com/bluecatengineering/limani/compare/v1.0.0...v1.1.0) (2024-06-21)
 
-
 ### Features
 
-* add View Logs to system header menu (ENG-52664) ([da34996](https://github.com/bluecatengineering/limani/commit/da34996b6175c4675e7362ef86e91274b315e619))
-* rename parameter contains bam to a generic name (ENG-54964) ([3353e1c](https://github.com/bluecatengineering/limani/commit/3353e1c69d7903e9803df4b4750ebd5ba169e27f))
-* **ui:** add a new UI component DropdownMenu (ENG-56423) ([6739192](https://github.com/bluecatengineering/limani/commit/6739192b1740ee4df28a703617bc8df448cbc2ce))
-* **ui:** add side panel ui component with two sizes (ENG-56435) ([277587d](https://github.com/bluecatengineering/limani/commit/277587d96bdf0f5687e0457a62d47d46e8ed575f))
-* update the new User Interface with recommendations made from the UX team (ENG-52668) ([1f1484e](https://github.com/bluecatengineering/limani/commit/1f1484e234e8897221b99ce4d9be6ac1d0a73a1d))
+-   add View Logs to system header menu (ENG-52664) ([da34996](https://github.com/bluecatengineering/limani/commit/da34996b6175c4675e7362ef86e91274b315e619))
+-   rename parameter contains bam to a generic name (ENG-54964) ([3353e1c](https://github.com/bluecatengineering/limani/commit/3353e1c69d7903e9803df4b4750ebd5ba169e27f))
+-   **ui:** add a new UI component DropdownMenu (ENG-56423) ([6739192](https://github.com/bluecatengineering/limani/commit/6739192b1740ee4df28a703617bc8df448cbc2ce))
+-   **ui:** add side panel ui component with two sizes (ENG-56435) ([277587d](https://github.com/bluecatengineering/limani/commit/277587d96bdf0f5687e0457a62d47d46e8ed575f))
+-   update the new User Interface with recommendations made from the UX team (ENG-52668) ([1f1484e](https://github.com/bluecatengineering/limani/commit/1f1484e234e8897221b99ce4d9be6ac1d0a73a1d))
```

Comment:
To successfully push the code and pass all automated tests, the code was formatted according to prettier's rules.